config/testdata/managed-fields/podinfo.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
```

Comment:
Small detail. It's better to start a yaml doc with `---`

Reply:
Flux YAML parser does not require files to start with a separator.